inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/transformation/relation/UnionNodeRelationTest.java

```diff
@@ -26,10 +26,15 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * Tests for{@link UnionNodeRelationTest}
+ * Tests for {@link UnionNodeRelationTest}
```

Comment:
`UnionNodeRelationTest` should be UnionNodeRelation

Reply:
done